DatadogInternal/Sources/Context/DeviceInfo.swift

```diff
@@ -96,4 +96,34 @@ extension DeviceInfo {
         #endif
     }
 }
+#else
```

Comment:
/suggestion I think it should be `#elseif macOS` because we are hard-coding `osName: "macOS"` in this case.

Reply:
the if is based on `#if canImport(UIKit)` because of `UIDevice` dependency, but I can add it for sanity